### DISCUSSION TITLE
I can't push an important file

### DISCUSSION THREAD

**Discussion by FilipScienski** [community/community#113321] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**What can I do?**

**I cannot push an important file due to the 100mb limit on GitHub.
 What can I do ?
 How to upload such a file?
 I need the answer by _yesterday_. 
Please and thank you for your help**

**Comment by deka27** [comment]
I would suggest using Git LFS.

Here is a helpful resource [link.](https://stackoverflow.com/questions/65820300/pushing-files-over-100mb-to-github)

**Reply by FilipScienski** [reply]
I tried it but it still is no cooperating with me

**Reply by deka27** [reply]
what is the error you are having while trying LFS, can you share that?

**Reply by FilipScienski** [reply]
Now I am writing code. i don't have time because project is until tomorrow. I will send it when i will have time

**Reply by deka27** [reply]
sure

**Reply by FilipScienski** [reply]
I think i fixed it idk how but at the moment works

**Reply by FilipScienski** [reply]
![aajaja](https://github.com/community/community/assets/133530486/a86a732b-b283-4dee-a326-1faf0c66666d)
